devops/makefiles/log.mk

```diff
@@ -0,0 +1,11 @@
+# Log
+# ---
+
+echo-cyan:  ## echo with cyan font https://gist.github.com/iamnewton/8754917
+	@echo -e "\e[36m${msg}\e[39m"
```

Comment:
Nice 💯 
Not sure why but it is not working in my terminal (`zsh 5.8 (x86_64-apple-darwin20.0)`).

Running `echo -e "\e[36mHello, World\e[39m"` works as expected but when I run `make test-all`, it just display `\e[36m2.1. Unit tests\e[39m` without coloring the test ^^

Reply:
aha, thanks @chrichri17 . I'm using a bash terminal. Color work on my side and on CI as well. Maybe you should use a bash terminal?